js/abp-preprocessed.es6.js

```diff
@@ -17,6 +17,8 @@ easylists = {
     }
 };
 
+trackerWhitelist = { };
```

Comment:
Is this a `var`? a `let`? a `const`? please delcare. remove whitespace between curly braces.